coverage.ini

```diff
@@ -0,0 +1,44 @@
+[run]
+branch = True
+source_pkgs =
+    acis_thermal_check,
+    acispy,
+    backstop_history,
+    prompt_toolkit,
+    Chandra,
+    Quaternion,
+    Ska,
```

Comment:
Is this getting all the Ska.* packages?  (And likewise for `Chandra`). Seems a little odd, might be better to call them out explicitly, but I don't feel strongly if this is working.

Reply:
Yes, this is actually a directory, I think. I would leave it.

Reply:
OK